### PR TITLE
[EHL] Clone FSP and Microcode from open source repo

### PR DESCRIPTION
**Pull request by jinjhuli**
1. Clone FSP from
https://github.com/intel/FSP/tree/master/ElkhartLakeFspBinPkg
2. Clone microcode from
https://github.com/tianocore/edk2-non-osi/tree/master/Silicon/Intel/ElkhartlakeSiliconBinPkg

Signed-off-by: jinjhuli <jin.jhu.lim@intel.com>

### CODE REVIEW

File: Silicon/ElkhartlakePkg/FspBin/FspBin.inf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ElkhartLakeFspBinPkg/FspBin/FSPRel.bin               : Silicon/ElkhartLakePkg/FspBin/FspDbg.bin
+  ElkhartLakeFspBinPkg/FspBin/FSPRel.bin               : Silicon/ElkhartLakePkg/FspBin/FspRel.bin
+  ElkhartLakeFspBinPkg/FspBin/FSP.bsf                  : Silicon/TigerlakePkg/FspBin/Fsp.bsf
+  ElkhartLakeFspBinPkg/Include/FspUpd.h                : Silicon/ElkhartLakePkg/Include/FspUpd.h
+  ElkhartLakeFspBinPkg/Include/FsptUpd.h               : Silicon/ElkhartLakePkg/Include/FsptUpd.h
+  ElkhartLakeFspBinPkg/Include/FspmUpd.h               : Silicon/ElkhartLakePkg/Include/FspmUpd.h
+  ElkhartLakeFspBinPkg/Include/FspsUpd.h               : Silicon/ElkhartLakePkg/Include/FspsUpd.h
+  ElkhartLakeFspBinPkg/Include/MemInfoHob.h            : Silicon/ElkhartLakePkg/Include/MemInfoHob.h
+  ElkhartLakeFspBinPkg/VbtBin/Vbt.bin                  : Platform/ElkhartLakeBoardPkg/VbtBin/Vbt.dat
+  ElkhartLakeFspBinPkg/VbtBin/Vbt.bsf                  : Platform/ElkhartLakeBoardPkg/VbtBin/Vbt.bsf
```
Comment (gdong1) [inline Silicon/ElkhartlakePkg/FspBin/FspBin.inf:26]: The build issue should be caused by the folder name.
"ElkhartLakePkg" should be "ElkhartlakePkg" in Silicon folder.

File: Silicon/ElkhartlakePkg/FspBin/FspBin.inf [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+[UserExtensions.SBL."CloneRepo"]
+  REPO    = https://github.com/intel/FSP.git
+  COMMIT  = 7954a8db808942a7430a00f4e5a757eb84951edc
+
+[UserExtensions.SBL."CopyList"]
+  ElkhartLakeFspBinPkg/FspBin/FSPRel.bin               : Silicon/ElkhartLakePkg/FspBin/FspDbg.bin
+  ElkhartLakeFspBinPkg/FspBin/FSPRel.bin               : Silicon/ElkhartLakePkg/FspBin/FspRel.bin
+  ElkhartLakeFspBinPkg/FspBin/FSP.bsf                  : Silicon/TigerlakePkg/FspBin/Fsp.bsf
```
Comment (myguitar) [inline Silicon/ElkhartlakePkg/FspBin/FspBin.inf:19]: TigerlakePkg?

File: Silicon/ElkhartlakePkg/Microcode/Microcode.inf [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+[Defines]
+  INF_VERSION          = 0x00010005
+  BASE_NAME            = Microcode
+  FILE_GUID            = 197DB236-F856-4924-90F8-CDF12FB875F3
+  MODULE_TYPE          = USER_DEFINED
+  VERSION_STRING       = 1.0
+
+[Sources]
+ m0190661_00000011.mcb
```
Comment (myguitar) [inline Silicon/ElkhartlakePkg/Microcode/Microcode.inf:17]: Indentation here - 2spaces.

**Review by myguitar — APPROVED** [review]
Looks good to me

File: Silicon/ElkhartlakePkg/FspBin/FspBin.inf (hunk 2 above)
Comment (jinjhuli) [reply Silicon/ElkhartlakePkg/FspBin/FspBin.inf]: Modified it, thanks for the review.

File: Silicon/ElkhartlakePkg/FspBin/FspBin.inf (hunk 1 above)
Comment (jinjhuli) [reply Silicon/ElkhartlakePkg/FspBin/FspBin.inf]: Updated, thanks for the review.

File: Silicon/ElkhartlakePkg/Microcode/Microcode.inf (hunk 3 above)
Comment (jinjhuli) [reply Silicon/ElkhartlakePkg/Microcode/Microcode.inf]: Updated, thanks for the review.

**Review by mauricema — APPROVED** [review]
Looks good.

**Review by gdong1 — APPROVED** [review]

**Pull request merged**